Resources/Prototypes/_White/Entities/Objects/Weapons/Guns/Turrets/ship_turrets.yml

```
@@ -0,0 +1,173 @@
+    drawdepth: Mobs
+    noRot: true
+    state: syndie_base
+  - type: DollyMixture
+  - type: CombatMode
+    isInCombatMode: true
+    toggleMouseRotator: false
+  - type: Damageable
+    damageContainer: Inorganic
+  - type: Destructible
+    thresholds:
+      - trigger:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)